src/fides/api/util/tcf/tc_string.py

```diff
@@ -180,3 +188,87 @@ def build_disclosed_vendors_string(model: TCModel) -> str:
     return base64.urlsafe_b64encode(
         _convert_bitstring_to_bytes(disclosed_vendor_bits)
     ).decode()
+
+
+def boolean_to_user_consent_preference(preference: bool) -> UserConsentPreference:
+    return UserConsentPreference.opt_in if preference else UserConsentPreference.opt_out
+
+
+FidesPreferenceType = Union[TCFPurposeSave, TCFVendorSave, TCFSpecialFeatureSave]
+
+
+def convert_to_fides_preference(
```

Comment:
This helper method transforms a section of the decoded TC string, into the format I'd expect when saving privacy preferences normally via the API